packages/components/src/navigable-container/container.tsx

```diff
@@ -115,9 +135,12 @@ class NavigableContainer extends Component {
 			return;
 		}
 
-		const context = getFocusableContext(
-			event.target.ownerDocument.activeElement
-		);
+		const { activeElement } = ( event.target as HTMLElement ).ownerDocument;
```

Comment:
Similarly to the previous comment, `event.target` could be `null` and therefore, accessing `ownerDocument.activeElement` would result in an error.

One alternative that came to mind is to use optional chaining:

```suggestion
		const activeElement = ( event.target as HTMLElement | null )
			?.ownerDocument?.activeElement;
```

Reply:
Good idea, committed in https://github.com/WordPress/gutenberg/pull/49377/commits/bff2e4b26e93e0f89f1c118a722c173c8a50b6e3